graphdatascience/model/model.py

```diff
@@ -20,7 +20,11 @@ def _query_prefix(self) -> str:
         pass
 
     def _list_info(self) -> DataFrame:
-        query = "CALL gds.beta.model.list($name)"
+        query = "CALL gds.model.list($name)"
+
+        if self._server_version < ServerVersion(2,5,0):
+            query = "CALL gds.beta.model.list($name)"
```

Comment:
```suggestion
        name_space = "beta." if self._server_version < ServerVersion(2,5,0) else ""
        query = f"CALL gds.{name_space}model.list($model_name) YIELD exists
```

propably need to format the code but its smaller and avoids reassigning ;)